github/actions_workflow_runs.go

```diff
@@ -94,6 +94,13 @@ type WorkflowRunAttemptOptions struct {
 	ExcludePullRequests *bool `url:"exclude_pull_requests,omitempty"`
 }
 
+// PendingDeploymentsRequest specificies body parameters to PendingDeployments
+type PendingDeploymentsRequest struct {
+	EnvironmentIDs *[]int  `json:"environment_ids"`
```

Comment:
I am passing integers here directly, I wonder if this needs to be wired to some other type.

Reply:
Yes, IDs are typically `int64`, so please change this to `[]int64` and remove the leading asterisk.

Reply:
Thanks! fixed.